javascript/src/test/pages/Messages/SearchResults.test.js

```diff
@@ -12,7 +12,7 @@ import userEvent from "@testing-library/user-event";
 //     };
 // });

```

Comment:
I wonder whether this commented out code should be removed.

Is there a good reason why it is still here?  If not, then please remove it.  If so, let me know why.